iguane.py

```diff
@@ -163,6 +163,13 @@ def fom(f):
 }
 
 
+VERSIONS = {
+    # v1.0 is equivalent to UGR
+    # https://docs.alliancecan.ca/wiki/Allocations_and_compute_scheduling
+    1.0: { 'fp16': 1.6, 'fp32': 1.6, 'tf32': 0.0, 'memgb': 0.8, 'membw': 0.0 },
```

Comment:
Curious to know why the weights are included in the version definition. I gather you want to offer some way to obtain the versions 1.0 weights for all future versions of Iguane ? I'm simply afraid this would amount to a lot of maintenance work where different versions of Iguane would have different versions of the weights. I guess my preconception of Iguane was that it should be the implementation of ONE version of the weights, and that it would be version tracked itself...

Reply:
The idea is to be able to compare different versions of UGR / IGUANE, particularly when we want to demonstrate the value of a new version of the weights compared to an older version. If the version of the code is linked to the version of IGUANE, wouldn't we need to uninstall reinstall the tool only to change the weights?

Reply:
The idea was that all that is required to define a new unit is a new function here that then gets automatically picked up if decorated with `@fom`:

https://github.com/mila-iqia/IGUANE/blob/master/iguane.py#L166-L188

A point-release could then simply change the default at here:

https://github.com/mila-iqia/IGUANE/blob/master/iguane.py#L215-L217